packages/flutter_adaptive_scaffold/lib/src/adaptive_scaffold.dart

```diff
@@ -526,11 +547,15 @@ class _AdaptiveScaffoldState extends State<AdaptiveScaffold> {
                 leading: widget.leadingExtendedNavRail,
                 trailing: widget.trailingNavRail,
                 selectedIndex: widget.selectedIndex,
-                destinations: widget.destinations
-                    .map((NavigationDestination destination) =>
-                        AdaptiveScaffold.toRailDestination(destination))
-                    .toList(),
+                destinations: destinations,
                 onDestinationSelected: _onDrawerDestinationSelected,
+                backgroundColor: navRailTheme.backgroundColor,
```

Comment:
Should we also include the changes to `backgroundColor` through `unselectedLabelTextStyle` in the changelog?